Leaf API.postman_collection.json

```diff
@@ -1421,6 +1026,631 @@
 								}
 							]
 						},
+						{
+							"name": "Provider Credentials",
+							"item": [
+								{
+									"name": "ClimateFieldView",
+									"item": [
+										{
+											"name": "Get credential of LeafUser",
```

Comment:
Lest include the provider name in the title of this endpoint

Reply:
Added it: https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/44/commits/8194d7cb4685389a79c930a74168de128fbfb605